cr/cmd/index.go

```diff
@@ -35,6 +37,15 @@ given GitHub repository's releases.
 		if err != nil {
 			return err
 		}
+
+		if len(config.ChartsRepo) > 0 {
+			fmt.Fprintf(os.Stderr, "ATTENTION: Flag --charts-repo is deprecated. It does not have any effect.\n"+
+				"The index.yaml is read from the '%s' branch instead.\n"+
+				"Loading index.yaml directly from the charts repository lead to problems as there is a delay between\n"+
+				"pushing to the github pages branch until things apear online.\n"+
+				"The flag will be removed with the next major release.", config.PagesBranch)
+		}
```

Comment:
I decided to print the deprecation message here. Alternative would have been the `releaser.UpdateIndexFile()` method.